### DISCUSSION TITLE
does this work on intraday data?

### DISCUSSION THREAD

**Discussion by kwtneo** [kernc/backtesting.py#366] [Q&A]
Hi, thanks for creating this - really cool. Had a question - how easy is it to adapt this to intraday data?

**Comment by Benouare** [comment]
Hi,
First, you must agregate all the intradays data that you want to work on,
After that, I think you just need to make "tick"( = dateTIME) unique. If it's the case you should be able to use bt. 

Cheers

**Comment by Zazle** [comment]
Hi,
If you haven't solve this problem, this is my solution below:
You can simply set the 'Date' index to "format='%Y/%m/%d %H:%M:%S'" to do intraday backtesting.
Good luck

**Reply by jsgaston** [reply]
or use klines for intraday
